hail/src/main/scala/is/hail/HailContext.scala

```diff
@@ -247,6 +247,12 @@ object HailContext {
     val hc = new HailContext(sparkContext, sqlContext, logFile, hailTempDir, branchingFactor)
     sparkContext.uiWebUrl.foreach(ui => info(s"SparkUI: $ui"))
 
+    var pipelineEmail = System.getenv("HAIL_PIPELINE_UPLOAD_EMAIL")
+    if (pipelineEmail == null)
+      pipelineEmail = sparkContext.getConf.get("hail.pipelineUploadEmail", null)
+    if (pipelineEmail != null)
+      hc.enablePipelineUpload(pipelineEmail)
```

Comment:
we should definitely warn here.

Reply:
I warn (used to info) in enablePipelineUpload.  Is that insufficient?

Reply:
Also, note, for some reason those warns are going into the log file but not the console, in spite of the fact they are running after configureLogging.  Still looking into it.